### DISCUSSION TITLE
Sentry mode alerts in the overview dashboard

### DISCUSSION THREAD

**Discussion by slavag** [teslamate-org/teslamate#3292] [Ideas]
Hi,
Is it possible to have a sentry mode alerts on the overview dashboard ?
Does this somehow exposed by tesla ?

Thanks

**Comment by cwanja** [comment]
There is no API endpoint today that delivers this value.

**Reply by vide** [reply]
There is no official Owners API dedicated endpoint but it can be inferred by the center_display_state (which is already published to mqtt) which can be used to infer if Sentry has detected (at least) one event. if `center_display_state` has value `7` it means that it is showing the "Sentry mode active" warning in the screen to bystanders. It would be great if this could be stored in the database.

**Reply by vide** [reply]
Actually the best solution would be to store the raw value every time it changes, with the timestamp of the change. that would already give enough flexibility in the dashboards to then show the data in the best way.

**Comment by dsz79** [comment]
There are other applications, such as Tessie, that can send notifications when a Sentinel event occurs. I assume they’re not using a Tesla-exclusive API, so there should be an endpoint in Tesla’s API to retrieve this data, right?

**Comment by zlymeda** [comment]
tesla telemetry has sentry mode field:
- SentryModeOff
- SentryModeIdle
- SentryModeArmed
- SentryModeAware
- SentryModePanic
- SentryModeQuiet
